### PR TITLE
Bug/side menu shows first

### PR DESCRIPTION
**Pull request by Kureev**
Fixes issue #107

@grabbou can you review this please? I think you can omit migration stuff and take a look only at index.js. If everything is fine, I'll merge it in.

### CODE REVIEW

**Comment by grabbou** [comment]
I think I was having this issue in my app yesterday, so gonna check it out today!

**Comment by grabbou** [comment]
I think if {menu} was rendered after contentView, we wouldn't have had that issue, but obviously that's how the things are working here.

LGTM, speaking cycle wise, should solve the problem addressed.

**Comment by Kureev** [comment]
Yeah, that's what I'm doing here :) Good.

File: index.js [hunk 1]
```diff
@@ -273,11 +280,18 @@ class SideMenu extends Component {
...
+    if (this.state.shouldRenderMenu) {
+      menu = <View style={styles.menu}>{this.props.menu}</View>;
+    }
+
     return (
       <View style={styles.container} onLayout={this.onLayoutChange.bind(this)}>
-        <View style={styles.menu}>
-          {this.props.menu}
-        </View>
+        {menu}
```
Comment (grabbou) [inline index.js]: Why not inline? Matter of taste, but I am not a huge fan of reassigning variables ;)
E.g.
```
{this.state.shouldRenderMenu && (
   <View style={styles.menu}>{this.props.menu}</View>
)}
```

Comment (Kureev) [inline index.js]: We can also do this. Make a fix for it and I'll :+1: it

Comment (grabbou) [inline index.js]: Ok, tomorrow morning we will be changing world :dash:

**Pull request merged**